x/lockup/keeper/iterator.go

```diff
@@ -49,71 +49,76 @@ func (k Keeper) iterator(ctx sdk.Context, prefix []byte) sdk.Iterator {
 }
 
 // LockIteratorAfterTime returns the iterator to get locked coins
-func (k Keeper) LockIteratorAfterTime(ctx sdk.Context, time time.Time) sdk.Iterator {
-	return k.iteratorAfterTime(ctx, types.KeyPrefixLockTimestamp, time)
+func (k Keeper) LockIteratorAfterTime(ctx sdk.Context, unlockingPrefix []byte, time time.Time) sdk.Iterator {
```

Comment:
Shouldn't these be `bool`s instead?  Callers of these functions shouldn't have to know what Prefixes and stuff it uses.

Reply:
Ok, bool makes more sense.